### PR TITLE
Updates selectors and logic for UI handler

### PR DESCRIPTION
**Pull request by StetsenTech**
**Ticket(s)**:
https://app.asana.com/0/1142638035116665/1164770882598457/f

**Description**:
Updates the selectors for updating the UI after a done button is click or retain data is finished loading.

Note: There are CSS changes that are needed to made this look correct. I will leave it here for reference.
```css

.show-answers input[data-correctness="correct"]:checked + label {
    color: #22B573;
}
.show-answers input[data-correctness="correct"]:not(:checked) + label {
    color: #B08833;
}
.show-answers input[data-correctness="incorrect"] + label {
    color: #EB3B46;
}
```

### CODE REVIEW

File: braven_custom.js [hunk 1]
```diff
@@ -325,18 +325,19 @@ function bzGiveVerboseFeedback(feedback, answerSpace, feedbackClass) {
 */
 var bzNewUiHandlers = {
   // Score a checklist question:
-  '.for-checklist' : function(){
+  '.module-block-checkbox .done-button,.module-block-key .done-button,.module-block-reflection .done-button' : function() {
```
Comment (sadleb) [inline braven_custom.js]: This logic used to be only for checklists, but now it's run for `.module-block-key` and `.module-block-reflection`. I don't know what those are, but I'm assuming they will always have a checklist in them and be graded the same and that's why the code mostly remains the same?

Comment (rshipp) [reply braven_custom.js]: i haven't had a chance to review this PR yet, but fyi any of the non-input-type-specific module-block-??? can have checklists.

**Comment by rshipp** [comment]
Same as @sadleb, this all looks good to me except the checklist part, which we might want to discuss first.

File: braven_custom.js (hunk 1 above)
Comment (StetsenTech) [reply braven_custom.js]: The short of this is it that it is used so that `show-answers` is applied to each option.

**Pull request merged**